src/Form/AkamaiConfigForm.php

```diff
@@ -128,6 +126,25 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#required' => TRUE,
     );
 
+    $form['devel_fieldset'] = array(
```

Comment:
I wouldn't mind seeing a verbose option that outlines operations in the operations class